specification/gp-connect-access-record-fhir.yaml

```diff
@@ -213,4 +253,119 @@ servers:
     description: Integration test environment.
   - url: 'https://api.service.nhs.uk/gp-connect-access-record-fhir'
     description: Production environment.
-paths: {}
+paths: 
+  /FHIR/STU3/$access-record-structured:
```

Comment:
Not sure this is consistent with the servers section above and the environments section